### ISSUE TITLE
builtin extensions are not working

### ISSUE THREAD

**Issue by bezreyhan** [palantir/tslint#1456]
### Bug Report

- __TSLint version__: `3.14.0`
- __TypeScript version__: `2.1.5`
- __Running TSLint via__:  `Webpack`

#### TypeScript code being linted

```ts
function hello(foo: string) {
  const str : string = 'bar' + foo;
  if (str)
    console.log('blah')
  console.log(str);
}

hello("foo")
```

with `tslint webpack` configuration:
```
tslint: {
    configuration: {
      "extends": "tslint:latest"
    }
}
```

#### Actual behavior

no warning are being reported

#### Expected behavior

warning should be printed in the terminal and console.

I tried both `tslint:latest` and `tslint:recommended` but neither worked. If I add the following to my config then I get an error for my curly braces: 
```
rules: {curly: true}
```

**Comment by bezreyhan** [comment]
This seems to be an issue with ts-loader. I'll closer this.
